package.json

```diff
@@ -102,6 +103,9 @@
     "ts-node": "^8.3.0",
     "tslint": "^5.19.0"
   },
+  "publishConfig": {
```

Comment:
What is this and why did you need to add it?

I’m not comfortable with using another registry other than npm.

Reply:
Sorry, this file is wrong..., I will remove it.